.github/workflows/create-quantic-package.yml

```diff
@@ -0,0 +1,57 @@
+name: Create Quantic Package Workflow
+
+env:
+  SFDX_AUTH_JWT_INSTANCE_URL: https://login.salesforce.com
+  SFDX_AUTH_JWT_USERNAME: sfdc.integration.devv2.hub@coveo.com
+  SFDX_AUTH_JWT_KEY: ${{ secrets.SFDX_AUTH_JWT_KEY_DEV }}
+  SFDX_AUTH_JWT_CLIENT_ID: ${{ secrets.SFDX_ATH_JWT_CLIENT_ID }}
+  PACKAGE_ID: 0Ho6g000000k9g8CAA
+
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: '30 17 * * *'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  create-and-promote-quantic-package:
+    name: Create and Promote Quantic Package
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    steps:
+      - name: Use Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: 14.x
+      - name: Check Out Repository
+        uses: actions/checkout@v2
+      - name: Install NPM dependencies
+        run: npm i -g sfdx-cli
+      - name: Create cache file
+        run: |
+          mkdir check-SHA
+          echo ${{ github.sha }} > github-sha.txt
+      - name: Check SHA
+        id: check_sha
+        uses: actions/cache@v1
+        with:
+          path: check-SHA
+          key: check-SHA-${{ github.sha }}
+      - name: Create jwt key file
+        if: steps.check_sha.outputs.cache-hit != 'true'
+        run: echo "$SFDX_AUTH_JWT_KEY" > server.key
+      - name: Connect to DevHub organization
+        if: steps.check_sha.outputs.cache-hit != 'true'
+        run: sfdx force:auth:jwt:grant --clientid $SFDX_AUTH_JWT_CLIENT_ID --jwtkeyfile server.key --username $SFDX_AUTH_JWT_USERNAME --setdefaultdevhubusername
+      - name: Create package version
+        if: steps.check_sha.outputs.cache-hit != 'true'
+        run: |
+          sfdx force:package:version:create --package $PACKAGE_ID --wait 30 -x --json >> package-version-result.json
+          cat package-version-result.json
+          PACKAGE_VERSION_ID="$(cat package-version-result.json | jq '.result.SubscriberPackageVersionId' -r)"
+          echo "$PACKAGE_VERSION_ID" > PACKAGE_VERSION_ID.txt
+          echo $PACKAGE_VERSION_ID
+          if [ $PACKAGE_VERSION_ID == "null" ]; then echo "Null Package ID" exit 1; fi
```

Comment:
I would do this check before printing to stdout with `echo`

Reply:
Also echo "Null Package ID" and exit 1 seem to be two different commands. Are you sure there isn't a ; missing in between?

Reply:
yes good catch, thanks